index.html

```diff
@@ -0,0 +1,171 @@
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <title>My capstone project</title>
+</head>
+
+<body>
+
+  <main class="main-page">
+    <section class="first-page">
+      <header class="toolbar-logout">
+        <nav>
+          <div class="logout">
+            <a class="fb" href=""><i class="fab fa-facebook light-grey"></i></a>
+            <a class="twit" href=""><i class="fab fa-twitter light-grey"></i></a>
+            <a class="logout-b" href="">Logout</a>
+            <a class="mypage" href="">Mypage</a>
+            <a class="tigrigna" href="">ትግርኛ</a>
+          </div>
+          <div class="toolbar">
+            <div class="image-a"><img src="assets/logo.png" alt="logo"></div>
+            <ul>
+              <li><a href="./about.html">About</a></li>
+              <li>Programs</li>
+              <li>Join</li>
+              <li>Partners</li>
+              <li>news</li>
+              <li>CC compaign</li>
+            </ul>
+          </div>
+          <div class="humberger" onclick="openNav()">
+            <i class='fa fa-bars menu-m'></i>
+          </div>
+          <div id="myNav" class="m-menu">
+            <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+            <div class="m-menu-content">
+
+              <a class="abt" href="./about.html" onclick="closeNav()">About</a>
+              <a hre="#speakers" onclick="closeNav()">Speakers</a>
+              <a href="#partners" onclick="closeNav()">Partners</a>
+              <a href="#" onclick="closeNav()">Join the Program</a>
+              <a href="#section-044" onclick="closeNav()">Contact</a>
+              <a href="#main-program" onclick="closeNav()">Main Program</a>
+
+
+            </div>
+          </div>
+        </nav>
+
+      </header>
+      <div class="agenda">
+        <div class="headers">
+          <h2 class="hello special-color">"Hello! Sharing world"</h2>
+          <h1 class="main-topic special-color">Group Engineering ENTREPRENEURSHIP & TECHNOLOGY CONFERENCE 2023
+          </h1>
+        </div>
+        <p class="introductory">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Et, illo! Rem cupiditate
+          nemo minima voluptate laboriosam deserunt, dignissimos quia incidunt id pariatur odit laborum sunt
+          reprehenderit labore blanditiis temporibus modi.
+        </p>
+        <div class="info">
+          <p class="date">
+            2023.06.28(MON) ~ 30(WED)
+          </p>
+          <p class="adress">@Hall of Mekelle institute of technology, Mekelle, Tigrai
+          </p>
+        </div>
+      </div>
+    </section>
+    <section class="main-bussiness">
+      <div class="main-hr">
+        <h2 class="main-p">Main Programs</h2>
+        <hr class="line-m">
+      </div>
+
+      <div class="programs">
+        <div class="consult">
+          <i class="fas fa-book-reader "></i>
+
+          <h3 class="bussiness-h1">Lecture</h3>
+          <p class="bussiness-para">
```

Comment:
- [ ] As suggested by the previous reviewer, Kindly make sure the content of your program section are `centered` in the middle of the page for a better look and feel of your page.
`Hint: you may consider saying:`
```css
.main-bussiness {
display: block;
width: 95%;
margin: some_value auto
}
```
![image](https://user-images.githubusercontent.com/97100091/227734023-65f9bc1e-0a80-462c-a6e5-387d2e5df3cc.png)